### ISSUE TITLE
Per-project projectdir settings ?

### ISSUE THREAD

**Issue by edouard-lopez** [yyjhao/markmon#10]
I edited my project settings to include the following code:

``` json
  "settings": {
    "projectdir": "/path/to/readme-directory"
  }
```

Without success.

How should I specify this setting on a per-project basis ?

**Comment by edouard-lopez** [comment]
This article may help: [Override plugin settings per-project via .sublime-project](http://sublimetext.userecho.com/topic/118670-override-plugin-settings-per-project-via-sublime-project/) as it seems to be the plugin's developer responsibility to include this feature.

**Comment by yyjhao** [comment]
Yes it currently does not support per project settings yet. I am too busy right now and I don't feel this feature is important enough. If you can send me a pull request I can review and merge it, otherwise I will have to leave this out for now.
